packages/openactive-broker-microservice/test/util/item-transforms-test.js

```diff
@@ -0,0 +1,102 @@
+const { expect } = require('chai');
+const {invertFacilityUseItems, createItemFromSubEvent} = require('../../src/util/item-transforms');
+
+describe('test/utils/item-transforms-test', () => {
+  describe('invertFacilityUseItems', () => {
+    it('should return the input items when no invertibleFacilityUseItems are present', () => {
+      const items = [
+        { data: { individualFacilityUse: null } },
+        { data: { individualFacilityUse: [] } },
+      ];
+      const result = invertFacilityUseItems(items);
+      expect(result).to.equal(items);
```

Comment:
.deep.equal ?

Otherwise, this will break if the function's logic changes in an inconsequential way